code/tests/utilities/helpers/test_AzureSearchHelper.py

```diff
@@ -134,116 +116,6 @@ def test_returns_search_client(
     assert search_client is search_client_mock.return_value
 
 
-@patch("backend.batch.utilities.helpers.AzureSearchHelper.SearchClient")
-@patch("backend.batch.utilities.helpers.AzureSearchHelper.SearchIndexClient")
-def test_creates_search_index_if_not_exists(
```

Comment:
I think we want to keep this test, but instead of the `# when` section just calling the constructor, it also now needs to call `get_client`